services/ingestion/etl/cordis/csv/README.md

```diff
@@ -0,0 +1,27 @@
+# CORDIS ETL mapping
+
+Model to compare with is available at: https://ec-europa.github.io/eubfr-data-lake/
+
+| Field                | Target                |
+| -------------------- | --------------------- |
+| rcn                  |                       |
+| id                   | project_id            |
+| acronym              |                       |
```

Comment:
This one looks like a candidate for a new field like `project_acronym`. In AGRI, we also have `Project acronym` which wasn't taken, and is sometimes available.